package.json

```
@@ -21,7 +21,7 @@
     "@opencloud-eu/prettier-config": "^1.0.0",
     "@opencloud-eu/tsconfig": "^1.0.0",
     "@opencloud-eu/web-test-helpers": "^1.0.0",
-    "@playwright/test": "^1.48.2",
+    "@playwright/test": "1.50.1",
```

Please keep the version range. Or did your remove it intentionally?

```suggestion
    "@playwright/test": "^1.50.1",
```